deploy/kubedirector/kubedirector.hpe.com_kubedirectorclusters_crd.yaml

```diff
@@ -147,6 +147,9 @@ spec:
                           ephemeral-storage:
                             type: string
                             pattern: '^([0-9.]+)([eEinumkKMGTP]*[-+]?[0-9]*)$'
+                  serviceAccountName:
+                        type: string
+                        minLength: 1
```

Comment:
We need to validation for this to make sure the serviceaccount exists in that namespace